### DISCUSSION TITLE
Where did the quality settings go?

### DISCUSSION THREAD

**Discussion by MikeyMan83** [pymedusa/Medusa#10185] [Q&A] [answered]
Hi All,

Just moved from a synology package to a docker image for medusa. However; somehow i cannot find the general quality settings anymore. Were they removed? I can only find them on show level...

Also seem to be unable to scan all files.

Hope someone can help me.

**Comment by p0psicles** [comment, marked as answer]
When you add a new show. You can save your configured quality settings as default.

**Reply by MikeyMan83** [reply]
Thanks, found that one, but no general settings anymore. By design i assume then?
